index.js

```diff
@@ -5,14 +5,36 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "orange"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const person = document.querySelector(`h2`)
+person.textContent= traveler.name
+const alias = document.querySelector(`em`)
+alias.textContent= traveler.nickname
+const pic = document.querySelector(`img`)
+pic.alt= traveler.name
+pic.src= traveler.photo
+
+
 
 
 /***** Deliverable 4 *****/
 
+const animal= createElement(`li`)
+
+traveler.animalSightings.forEach (function(sighting) {
+    animal.innerHTML += `<li data-id = "${sighting.id}">
+    <p>"${sighting.description}"</p>
+    <img src= "${sighting.photo}" alt= "${sighting.species}"/>
+    <a href="${sighting.link}" target="_blank">Here's a video about the "${sighting.species}" species!</a>
+  </li>`
+})
+
+const animalsU1 = document.querySelector('ul#animals');
+
+animalsUl.append(animal);
```

Comment:
You're on the right track! Currently you're creating an `<li>` and then adding a bunch of other `<li>`s within the `<li>`, so the parent of all the animal `<li>`s is another `<li>`.  You need to create [an `<li>`](https://www.w3schools.com/tags/tag_li.asp) for _each_ animal sighting post and the parent should be the `animalsUl`. You have several options; here are two: 

- Change the `innerHTML` of the `animalsUl` instead
```js
const animalsUl = document.querySelector('ul#animals');

traveler.animalSightings.forEach( function(sighting) {
    animalsUl.innerHTML += `<li data-id="${sighting.id}">
    <p>${sighting.description}</p>
    <img src="${sighting.photo}" alt="${sighting.species}"/>
    <a href="${sighting.link}" target="_blank">Here's a video about the ${sighting.species} species!</a>
    </li>`
});
```

- Using [document fragment](https://developer.mozilla.org/en-US/docs/Web/API/Document/createDocumentFragment) (more advanced)
```js
const fragment = document.createDocumentFragment();;

traveler.animalSightings.forEach( function(sighting) {
    fragment.innerHTML += `<li data-id="${sighting.id}">
    <p>${sighting.description}</p>
    <img src="${sighting.photo}" alt="${sighting.species}"/>
    <a href="${sighting.link}" target="_blank">Here's a video about the ${sighting.species} species!</a>
    </li>`
});

const animalsUl = document.querySelector('ul#animals');
animalsUl.append(fragment);
```